ocs_ci/ocs/platform_nodes.py

```
@@ -4249,3 +4262,504 @@ def disable_nodes_network_temporarily(self, nodes, duration=20):
+        Args:
+            nodes (list): The OCS objects of the nodes to restart
+            force (bool): Force the restart operation
+
+        """
+        logger.info(
+            f"Restarting agent nodes by stop and start: {[n.name for n in nodes]}"
+        )
+
+        for node in nodes:
+            self._power_operation(node.name, "off", force=force)
+            logger.info(f"Successfully stopped node: {node.name}")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)